Add-to-app iOS flutter_post_install post_install documentation (#7032)
flutter/website: src/development/add-to-app/ios/project-setup.md

<li markdown="1">

> In the `Podfile` `post_install` block, call `flutter_post_install(installer)`.

Reviewer: ```suggestion
In the `Podfile`'s `post_install` block, call `flutter_post_install(installer)`.
```

<!--code-excerpt "MyApp/Podfile" title-->
```ruby
post_install do |installer|
  flutter_post_install(installer) if defined?(flutter_post_install)
end
```

{{site.alert.note}}

>   The `flutter_post_install` method has been recently added to Flutter, and
>   adds build settings to support native Apple Silicon `arm64` iOS simulators.
>   Include `if defined?(flutter_post_install)` to ensure your `Podfile`
>   is valid if you are running on older versions of Flutter that do not have this method.

Reviewer: ```suggestion
  The `flutter_post_install` method (recently added to Flutter),
  adds build settings to support native Apple Silicon `arm64` iOS simulators.
  Include the `if defined?(flutter_post_install)` check to ensure your `Podfile`
  is valid if you are running on older versions of Flutter that don't have this method.
```
